### PR TITLE
Figure 4.3.0

### PR DESCRIPTION
**Pull request by jburel**
* Add entry for 4.3.0
* Add missing entry for 4.2.0

cc @will-moore

### CODE REVIEW

File: _posts/2020-05-14-figure-4-3-0.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Release of OMERO.figure 4.3.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.figure 4.3.0
+---
+
+We are pleased to announce the release of OMERO.figure 4.3.0.
+
+- Add support for time-stamps in milliseconds ([#378](https://github.com/ome/omero-figure/pull/378))
```
Comment (mtbc) [inline _posts/2020-05-14-figure-4-3-0.md:9]:
```suggestion
- Add support for timestamps in milliseconds ([#378](https://github.com/ome/omero-figure/pull/378))
```
to match timestamps below.

**Comment by will-moore** [comment]
Great, thanks 👍

File: omero/figure/index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@ <h1 class="hero-main-header"><img src="{{ site.baseurl }}/img/logos/omero-figure
                     <h4>Create quality figures<br>from your OMERO images</h4>
                     <br>
                     <a href="https://pypi.python.org/pypi/omero-figure/" class="large btn-red button sites-button" style="text-shadow: none;">Install OMERO.<span class="lowercase">f</span>igure</a>
-                    <a href="{{ site.baseurl }}/2019/10/28/figure-4-1-0.html" class="hero-link-dark" style="text-shadow: none;">Read the Release Announcement</a>
+                    <a href="{{ site.baseurl }}/2020/05/15/figure-4-3-0.html" class="hero-link-dark" style="text-shadow: none;">Read the Release Announcement</a>
```
Comment (sbesson) [inline omero/figure/index.html:25]: Are we planning to ship a software release today (i.e. on Friday) or better to wait till Monday?

Comment (will-moore) [reply omero/figure/index.html:25]: I think we are OK to release figure today. It's a minor release and is unlikely to get lots of installs and bugs reported over the weekend.
Would be good to get it out of the way.

Comment (jburel) [reply omero/figure/index.html:25]: I think we are fine to push it today, considering everything we have on our plate at the moment

**Comment by jburel** [comment]
4.3.0 is now available on pypi. Merging

**Pull request merged**